packages/ui-alerts/src/Alert/__tests__/Alert.test.tsx

```diff
@@ -174,7 +174,10 @@ describe('<Alert />', async () => {
         </Alert>
       )
 
-      expect(subject.getDOMNode()).to.not.exist()
+      const root = document.querySelector('[data-ui-test-utils]')!
+
+      expect(root.children.length).to.equal(0)
+      expect(liver.children.length).to.equal(1)
     })
```

Comment:
I had to change this test because the component will return the result of `React.createPortal` when the `screenReaderOnly` prop is true and thus the `subject.getDOMNode()` would return the rendered content in the Portal.
Instead it will check whether the root of the React tree is empty or not.